packages/EasyCodingStandardTester/src/Testing/IntegrationTestCaseTrait.php

```diff
@@ -14,7 +14,7 @@ trait IntegrationTestCaseTrait
     /**
      * @var string
      */
-    private $splitLine = '#-----\n#';
+    private $splitLine = '#-----' . PHP_EOL . '#';
```

Comment:
Better reuse `AbstractCheckerTestCase::SPLIT_LINE` and make it public

Reply:
Done. I also thought about reusing it in ExplicitToAutodiscoveryConverterTest::SPLIT_LINE but the number of "-" isn't the same.

When I started replacing --- by ----- in yaml files, PHPStorm started to say "services" key was duplicated. It only tells that when the number of "-" is different from 3. Was that the reason behind this specific number of "-"?

If we want to reuse the const, we could replace ----- by --- everywhere else, what do you think?

Reply:
It should be constant value: `-----`

If it's something else, it's a bug

Reply:
Ok, I will replace it.

Reply:
Done. However, I find strange having to import something from packages/EasyCodingStandardTest to packages/Autodiscovery. Don't you think the constant should be relocated somewhere global?

Reply:
I didn't notice that, see:
https://github.com/Symplify/Symplify/pull/1650#discussion_r344513809